sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -840,72 +865,130 @@ struct save_log : detail::run_time_property_key<detail::PropKind::BuildLog> {
 using save_log_key = save_log;
 
 template <>
-struct is_property_key_of<save_log_key,
-                          sycl::kernel_bundle<bundle_state::ext_oneapi_source>>
+struct is_property_key_of<save_log_key, detail::build_source_bundle_props>
     : std::true_type {};
 
 /////////////////////////
-// syclex::is_source_kernel_bundle_supported
+// PropertyT syclex::registered_kernel_names
 /////////////////////////
+struct registered_kernel_names
```

Comment:
According to the spec, this should also contain methods:
`registered_kernel_names();` and `void add(const std::string &name)`. Any reason not to include them here?

Reply:
good catch. thanks